libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/extension/AuditExtensionMitigationType.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.di.ipv.core.library.auditing.extension;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record AuditExtensionMitigationType(@JsonProperty("mitigation_type") String mitigationType)
```

Comment:
Our other extensions are excluded from coverage, which seems reasonable - in general I don't see much value in unit testing model classes.